crates/turborepo-ui/src/color_selector.rs

```diff
@@ -38,14 +38,10 @@ impl ColorSelector {
             return style;
         }
 
-        let color = {
-            self.inner
-                .write()
-                .expect("lock poisoned")
-                .insert_color(key.to_string())
-        };
-
-        color
+        self.inner
```

Comment:
I'm a little iffy on this one, does this alter when the lock gets released at all? Just wondering because the code was clearly written to ensure that the lock got released as soon as the color was inserted.